src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,29 @@
+package duke.commands;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Abstract class which contains the 2 functions all classes that
+ * inherit from the Command Class must have.
+ *
+ * @author Cheam Jia Wei
+ */
+public abstract class Command {
+    /**
+     * Executes what the Command is supposed to do.
+     *
+     * @param taskList The TaskList that will be modified or accessed.
+     * @param inter The Ui that will interact with the user.
+     * @param store The storage that will help store the task into the data file if TaskList is modified.
+     */
```

Comment:
Might be better to include @return for execute method, not obvious at first glance that the return value is the message to be printed